bin/postinstall

```diff
@@ -40,6 +40,15 @@ ${CLI} config:set SECRET_TOKEN="$secret_token"
 # migrate
 ${CLI} run rake ${rake_commands} || true
 
+# migrate previous repositories with reposman to managed
+SVN_REPOSITORIES = $(${CLI} config:get SVN_REPOSITORIES)
+if [ -z "$SVN_REPOSITORIES" ]; then
+	${CLI} run rake scm:migrate:managed "file://${SVN_REPOSITORIES}"
```

Comment:
Shouldn't it be `${CLI} run rake scm:migrate:managed["file://${SVN_REPOSITORIES}"]`.